src/quo2/components/loaders/skeleton/view.cljs

```diff
@@ -0,0 +1,46 @@
+(ns quo2.components.loaders.skeleton.view
+  (:require [quo2.theme :as theme]
+            [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.loaders.skeleton.style :as style]
+            [quo2.components.loaders.skeleton.constants :as constants]))
+
+(defn- skeleton-item
+  [index content color]
+  [rn/view {:style (style/container content)}
+   [rn/view {:style (style/avatar color)}]
+   [rn/view {:style style/content-container}
+    [rn/view
+     {:style (style/content-view
+              {:type    (if (= content :list-items) :message :author)
+               :index   index
+               :content content
+               :color   color})}]
+    [rn/view
+     {:style (style/content-view
+              {:type    (if (= content :list-items) :author :message)
+               :index   index
+               :content content
+               :color   color})}]
+    (when (= content :notifications)
+      [rn/view
+       {:style (style/content-view {:type    :message2
+                                    :index   index
+                                    :content content
+                                    :color   color})}])]])
+
+(defn internal-view
```

Comment:
Small Improvement: Updating `internal-view` as private would prevent using it without theme context.